gno.land/pkg/sdk/vm/keeper_test.go

```diff
@@ -536,6 +640,68 @@ func main() {
 	assert.Equal(t, expectedString, res)
 }
 
+func TestVMKeeperRunImportDraft(t *testing.T) {
+	env := setupTestEnv()
+	ctx := env.vmk.MakeGnoTransactionStore(env.ctx)
+
+	// Give "addr1" some gnots.
+	addr := crypto.AddressFromPreimage([]byte("addr1"))
+	acc := env.acck.NewAccountWithAddress(ctx, addr)
+	env.acck.SetAccount(ctx, acc)
+
+	// Create test package.
+	const pkgPath = "gno.land/r/test"
+	files := []*std.MemFile{
+		{
+			Name: "gnomod.toml",
+			Body: `module = "gno.land/r/test"
+gno = "0.9"
+draft = true`,
+		},
+		{
+			Name: "test.gno",
+			Body: `package test
+
+func Echo(cur realm) string {
+	return "hello world"
+}`,
+		},
+	}
+	ctx = ctx.WithBlockHeader(&bft.Header{ChainID: "test-chain-id", Height: 0})
+	msg1 := NewMsgAddPackage(addr, pkgPath, files)
+	err := env.vmk.AddPackage(ctx, msg1)
+	assert.NoError(t, err)
+
+	ctx = ctx.WithBlockHeader(&bft.Header{ChainID: "test-chain-id", Height: 42})
+
+	// Msg Run Echo function.
+	coins := std.MustParseCoins("")
+	files = []*std.MemFile{
+		{
+			Name: "main.gno",
+			Body: `
+package main
+
+import "gno.land/r/test"
+
+func main() {
+	msg := test.Echo(cross)
+	println(msg)
+}
+`,
+		},
+	}
+	msg2 := NewMsgRun(addr, coins, files)
+	_, err = env.vmk.Run(ctx, msg2)
+	assert.Error(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Oh, actually `maketx run` should be able to import drafts!

Reply:
i changed the type check mode to one mode that act like strict for gnomod rules & block draft import
this mode is used only in add pkg (not in run msg) if not genesis time
4eb5b20